### PR TITLE
dev/how-to/maintain: update Usage Examples in slow query 

### PR DESCRIPTION
**Pull request by huangxiuyan**
Via: https://github.com/pingcap/docs-cn/pull/1278
@crazycs520 @lilin90 PTAL, thanks!

### CODE REVIEW

File: dev/how-to/maintain/identify-slow-queries.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ The slow query log is a record of SQL statements that took a long time to perfor
 
 The slow log format is updated in TiDB v2.1.8 and later. For the slow query log information in versions earlier than v2.1.8, see [this file](https://github.com/pingcap/docs/blob/master/v2.1/sql/slow-query.md).
 
+> **Note:**
+>
+> There are very few field information of slow query log in TiDB 2.1.8 version. The field information mentioned in the following part is added later in TiDB 3.0.0 version.
```
Comment (CaitinChen) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
> There is very few field information of slow query logs in TiDB v2.1.8. Some of the field information mentioned in the following part is added in TiDB v3.0.0.
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 2]
```diff
@@ -43,19 +46,18 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
 * `Txn_start_ts`: The start timestamp of the transaction (transaction ID). You can use this value to `grep` the transaction-related logs.
 * `User`: The name of the user who executes this statement.
 * `Conn_ID`: The Connection ID (session ID). For example, you can use the keyword `con:3` to `grep` the log whose session ID is 3.
-* `Query_time`: Indicates the execution time of this statement. Only the statement whose execution time exceeds slow-threshold outputs this log (the unit is second). The unit of all the following time fields is second.
+* `DB`: The current database.
+* `Index_ids`: The IDs of the indexes involved in the statement.
+* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, such as `analyze`, `load variables`, etc. `false` indicates the SQL statement is executed by the user.
```
Comment (CaitinChen) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, such as `analyze`, `load variables`, etc. `false` indicates that the SQL statement is executed by the user.
```

Comment (CaitinChen) [reply dev/how-to/maintain/identify-slow-queries.md]: Your expression is right. But in technical writing, it is recommended to add "that" here.

File: dev/how-to/maintain/identify-slow-queries.md [hunk 3]
```diff
@@ -43,19 +46,18 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
...
+* `Digest`: The fingerprint of the SQL statement.
+* `Memory_max`: Indicates the maximum memory space used during the execution period of this SQL statement (the unit is byte).
```
Comment (CaitinChen) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Memory_max`: The maximum memory space used during the execution period of this SQL statement (the unit is byte).
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 4]
```diff
@@ -43,19 +46,18 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
...
+* `Query_time`: 16.479155653
```
Comment (CaitinChen) [inline dev/how-to/maintain/identify-slow-queries.md]: Why did you update this line?

File: dev/how-to/maintain/identify-slow-queries.md [hunk 5]
```diff
@@ -76,33 +78,33 @@ tidb > show create table INFORMATION_SCHEMA.SLOW_QUERY;
...
+|            |   `Backoff_time` double unsigned DEFAULT NULL,              |
+|            |   `Request_count` bigint(20) unsigned DEFAULT NULL,         |
+|            |   `Total_keys` bigint(20) unsigned DEFAULT NULL,            |
+|            |   `Process_keys` bigint(20) unsigned DEFAULT NULL,          |
 |            |   `DB` varchar(64) DEFAULT NULL,                            |
 |            |   `Index_ids` varchar(100) DEFAULT NULL,                    |
-|            |   `Is_internal` tinyint(1) UNSIGNED DEFAULT NULL,           |
+|            |   `Is_internal` tinyint(1) unsigned DEFAULT NULL,           |
 |            |   `Digest` varchar(64) DEFAULT NULL,                        |
 |            |   `Query` varchar(4096) DEFAULT NULL                        |
```
Comment (CaitinChen) [inline dev/how-to/maintain/identify-slow-queries.md]: You missed Line 92 - Line 101 in https://github.com/pingcap/docs-cn/pull/1278/files.

**Comment by huangxiuyan** [comment]
@CaitinChen PTAL, thanks!

File: dev/how-to/maintain/identify-slow-queries.md (hunk 5 above)
Comment (huangxiuyan) [reply dev/how-to/maintain/identify-slow-queries.md]: Addressed, thanks for reminding me about it.

File: dev/how-to/maintain/identify-slow-queries.md (hunk 4 above)
Comment (huangxiuyan) [reply dev/how-to/maintain/identify-slow-queries.md]: Sorry, I confused it with that in the example above.

**Comment by CaitinChen** [comment]
@crazycs520 PTAL

**Pull request merged**